sources/web/datalab/polymer/components/table-inline-details/table-inline-details.html

```diff
@@ -53,6 +53,7 @@
       <paper-spinner active hidden$={{!_busy}}></paper-spinner>
     </div>
     <div id="container" hidden$={{!_table}}>
+      <span hidden$=[[!_tableMessage]]>[[_tableMessage]]</span>
```

Comment:
Do we actually need the `hidden` attribute here? If there's no message, it's an empty span, which is inlined anyway right?

Reply:
I think the intent is more clear this way.